packages/perseus/src/widgets/radio/radio.ff.tsx

```diff
@@ -24,6 +25,16 @@ type State = {
     choiceStates: ChoiceStateWithoutSelected[];
 };
 
+function initChoiceStates(choices: Props["choices"]) {
+    return choices.map(() => ({
+        highlighted: false,
```

Comment:
I thought `highlighted` was removed recently? @SonicScrewdriver Was that something you were working on?

Reply:
Looks like it hasn't been removed yet, so it makes sense to keep it here for now

Reply:
We can't remove it yet as it's part of the ChoiceState iirc, which is used by the old radio widget. 

However, it's some kind of zombie property that doesn't seem to have been used in a LONG time, and we are never setting it to true.